src/SFA.DAS.Provider.PR.Web/Views/EmployerDetails/Index.cshtml

```diff
@@ -9,7 +9,8 @@
         <div class="govuk-!-width">
             <h1 class="govuk-heading-l govuk-!-margin-bottom-6">Employer account details</h1>
             <h2 class="govuk-heading-m govuk-!-margin-bottom-1">@Model.AccountLegalEntityName</h2>
-            <p class="govuk-body govuk-!-margin-bottom-8"><strong>Agreement ID: </strong>@Model.AccountLegalEntityPublicHashedId</p>
+            <p asp-show="!@Model.ShowAgreementId" class="govuk-body govuk-!-margin-bottom-8"></p>
+            <p asp-show="@Model.ShowAgreementId" class="govuk-body govuk-!-margin-bottom-8"><strong>Agreement ID: </strong>@Model.AccountLegalEntityPublicHashedId</p>
```

Comment:
Not sure if we need to check if value is empty here

Reply:
On the design, if there isn't an agreement id the subtitle isn't shown

Reply:
So empty para is still not needed right?

Reply:
No, I'll be removing this